esmvaltool/diag_scripts/portrait_plot.py

```diff
@@ -0,0 +1,505 @@
+    ('mean', 'median', 'centered_mean', 'centered_median', None).
+    Divide by median or mean if not None. Subtract median/mean afterwards if
+    centered.
+    By default 'centered_median'.
+x_by: str, optional
+    Metadata key for x coordinate.
+    By default 'alias'.
+y_by: str, optional
+    Metadata key for y coordinate.
+    By default 'variable_group'.
+group_by: str, optional
+    Metadata key for grouping.
+    Grouping is always applied in x direction. Can be set to None to skip
+    grouping into subplots.
+    By default 'project'.
+split_by: str, optional
+    The rectangles can be split into 2-4 triangles. This is used to show
+    metrics for different references. For this case there is no need to change
+    this parameter. Multiple variables can be set in the recipe with `split`
+    assigned as extra_facet to label the different references. Data without
+    a split assigned will be plotted as main rectangles, this can be changed
+    by setting default_split parameter.
+    By default 'split'.
+default_split: str, optional
+    Data labeled with this string, will be used as main rectangles. All other
+    splits will be plotted as overlays. This can be used to choose the base
+    reference, while all references are labeled for the legend. If None, the
+    first split will be used as default.
+    By default None.
+plot_legend: bool, optional
+    If True, a legend is plotted, when multiple splits are given.
+    By default True.
+legend: dict, optional
+    Customize, if, how and where the legend is plotted. The 'best' position
+    and size of the legend depends on multiple parameters of the figure
+    (i.e. lengths of labels, aspect ratio of the plots...). Might require
+    manual adjustment of `x`, `y` and `size` to fit the figure layout.
+    Keys (each optional) that will be handled are:
+
+    position: str or None, optional
+        Position of the legend. Can be 'right' or 'left'.
+        Or set to None to disable plotting the legend. By default 'right'.
+    x_offset: float, optional
+        Manually adjust horizontal position to save space or fix overlap.
+        Number given in Inches. By default 0.
+    y_offset: float, optional
+        Manually adjust vertical position to save space or fix overlap.
+        Number given in Inches. By default 0.
+    size: float, optional
+        Size of the legend in Inches. By default 0.3.
+
+plot_kwargs: dict, optional
+    Dictionary that gets passed as kwargs to `matplotlib.pyplot.imshow()`.
+    Colormaps will be converted to 11 discrete steps automatically. Default
+    colormap RdYlBu_r and limits vmin=-0.5, vmax=0.5 can be changed using
+    keywords like: cmap, vmin, vmax.
+    By default {}.
+cbar_kwargs: dict, optional
+    Dictionary that gets passed to `matplotlib.pyplot.colorbar()`.
+    E.g. label, ticks...
+    By default {}.
+plot_properties: dict, optional
+    Dictionary that gets passed to `matplotlib.axes.Axes.set()`.
```

Comment:
```suggestion
    Dictionary that gets passed to :meth:`matplotlib.axes.Axes.set`.
```

This will create a link. Applies not only to this line.